tests/e2e/19_private_keys.spec.js

```diff
@@ -0,0 +1,63 @@
+const TestUtil = require('../utils/TestUtils')
+const OverviewPage = require('../Pages/OverviewPage')
+const HomePage = require('../Pages/HomePage')
+const PasswordPage = require('../Pages/PasswordPage')
+const puppeteer = require('puppeteer')
+const { expect } = require('chai')
+
+const testUtil = new TestUtil()
+const overviewPage = new OverviewPage()
+const homePage = new HomePage()
+const passwordPage = new PasswordPage()
+
+let browser, page
+const password = '123123123'
+
+describe('Private key exports-[mainnet,smoke]', async () => {
+  beforeEach(async () => {
+    browser = await puppeteer.launch(testUtil.getChromeOptions())
+    page = await browser.newPage()
+    await page.goto(testUtil.extensionRootUrl, { waitUntil: 'load', timeout: 60000 })
+    await homePage.ScrollToEndOfTerms(page)
+    await homePage.ClickOnAcceptPrivacy(page)
+
+    // Import wallet option
+    await homePage.ClickOnImportWallet(page)
+    // Enter seed words and submit
+    await homePage.EnterSeedWords(page)
+    // Create a password & submit
+    await passwordPage.SubmitPasswordDetails(page, password)
+    // overview page
+    await overviewPage.HasOverviewPageLoaded(page)
+    await overviewPage.CloseWatsNewModal(page)
+    if (process.env.NODE_ENV === 'mainnet') {
+      await overviewPage.SelectNetwork(page, 'mainnet')
+    } else {
+      await overviewPage.SelectNetwork(page)
+    }
+  })
+  afterEach(async () => {
+    await browser.close()
+  })
+
+  it('BTC export private key', async () => {
+    await overviewPage.SelectChain(page, 'BTC')
+    await page.waitForSelector('#BTC_swap_button', { visible: true })
+    // Click on Export Private Key
+    await overviewPage.ClickOnBurgerIcon(page)
+    await page.waitForSelector('#export_privkey', { visible: true })
+    await page.click('#export_privkey')
+    await page.waitForSelector('#i_have_privacy_button', { visible: true })
+    await page.click('#i_have_privacy_button')
+    await page.waitForSelector('#password', { visible: true })
+    await page.type('#password', password)
+    await page.click('#checkbox')
+    await page.click('#continue_button_to_see_seed_phrase')
+    // Private key screen
+    await page.waitForSelector('#private-key-textarea', { visible: true })
+    const privateKeyTextArea = await page.$eval('#private-key-textarea', el => el.getAttribute('readonly'))
+    expect(privateKeyTextArea).equals('readonly')
```

Comment:
maybe also expect that the textarea contains a string that looks like a privatekey, eg something other than `n/a` ?

Reply:
yes trying to get the textContent but its empty
![image](https://user-images.githubusercontent.com/5720338/142678805-4f5acbc6-a8ad-40f1-ae12-fcb6161e3b55.png)

Reply:
` const privateKey = await page.$eval('#private-key-textarea', el => el.value)
    expect(privateKeyTextArea).equals('readonly')
    expect(privateKey, 'Private key export value shouldn\'t be n/a')
      .to.not.contain.oneOf(['n/a', 'NaN', null])`

Reply:
![image](https://user-images.githubusercontent.com/5720338/142679683-563ea096-9e58-4826-acb0-f2d353728cd0.png)